integrate/storage/python-client.md

```diff
@@ -61,9 +61,6 @@ The above command will import the contents of the `coords.csv` file into the new
 also mark the `id` column as the primary key.
 
```

Comment:
```suggestion
### Example --- Load to existing table, incrementally

To load data incrementally into an existing table, we can use the [load](https://github.com/keboola/sapi-python-client/blob/5a93926c2191ccd6b7402c9e24d9912884d87d4c/kbcstorage/tables.py#L207) method, where `table_id` is the ID of the table that you want to load into, and `path` is the path to your csv file containing the data:

{% highlight python %}

from kbcstorage.client import Client

client = Client('https://connection.keboola.com', 'your-token')

client.tables.load(table_id=table_id, file_path=path, is_incremental=True)

{% endhighlight %}
```
Maybe we can add a simple example?

Reply:
thanks, it much better. I hope it works :)